controllers/cluster_for_akodeploymentconfig_intg_test.go

```diff
@@ -0,0 +1,158 @@
+// Copyright 2022 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package controllers_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	clusterv1 "sigs.k8s.io/cluster-api/api/v1beta1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	akoov1alpha1 "github.com/vmware-samples/load-balancer-operator-for-kubernetes/api/v1alpha1"
+	"github.com/vmware-samples/load-balancer-operator-for-kubernetes/pkg/test/builder"
+	testutil "github.com/vmware-samples/load-balancer-operator-for-kubernetes/pkg/test/util"
+)
+
+func intgTestClusterDisableAVIWithoutAnyADC() {
+	var (
+		ctx           *builder.IntegrationTestContext
+		staticCluster *clusterv1.Cluster
+	)
+
+	BeforeEach(func() {
+		ctx = suite.NewIntegrationTestContext()
+		staticCluster = testutil.GetDefaultCluster()
+	})
+
```

Comment:
Could we add two more test cases:
- default adc with empty selector -> create two clusters, both should be selected by default adc
- default adc with non-empty selector -> create a cluster selected by default adc -> create a new adc also can select cluster -> expect cluster should not change the adc

Reply:
added